test/test_experience_replay.py

```diff
@@ -0,0 +1,16 @@
+from src.experience_replay import ReplayMemory
+import numpy as np
+import random
+    
+#Populate experience buffer
+def test_experience_replay():
+    mem = ReplayMemory(10000, 500)
+    for i in range(0,500):
+        frame = np.random.randint(255, size=(240, 256))
+        action = np.random.randint(4)
+        reward = np.random.randint(2)
+        terminal = np.random.choice(a=[False, False, False, False, False, False, False, False, True])
+        
+        mem.add(action, reward, frame, terminal)
+    batch = mem.getMinibatch
+    assert(batch)
```

Comment:
Hmm I'm not entirely sure what this test is doing. It seems like it is just asserting that mem has a method getMinibatch. Could you fix this and then also add a brief comment above the assertion to describe the intended  assertion.